src/deploy/osp_deployer/director.py

```diff
@@ -865,6 +865,17 @@ def deploy_overcloud(self):
                                     " --ntp " + \
                                     self.settings.sah_node.provisioning_ip
 
+        if self.settings.hpg_enable is True:
+            cmd += " --enable_hugepage "
```

Comment:
nit: should hugepage be plural hugepages?

Reply:
Changed to plural.